packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -63,7 +65,7 @@ const unpublishGranule = async (knexOrTransaction, granule) => {
         published: false,
         cmr_link: undefined,
       },
-      ['cumulus_id']
+      ['cumulus_id', 'published']
```

Comment:
Interesting -- I see why this was. added in deleteGranulesAndFiles... was this fixing an integration test?

Reply:
This is added here so that we can remove a previous `pgGranule.get()` call. After some refactoring the only thing that `get` was giving us that we didn't already have was `published`. This change just makes the `update` method return that field so we can eliminate the `get`. This was the commit:

4b006bd0b60a9c8fc412c8e66795ea5a61184280

Reply:
Yeah, it was either an integration test or a unit test that started failing after the most recent refactors. Either way, the update here needs to return `published` because this would be used for `deleteGranuleAndFiles` if we got that far. 

The test I was working on for `bulkGranuleDelete when forceRemove is true` started throwing `You can't delete a published granule', indicating I wasn't passing this properly.